src/numeric_index.h

```diff
@@ -53,7 +53,7 @@ typedef struct rtNode {
 } NumericRangeNode;
 
 typedef struct {
-  size_t sz;
+  int sz;
```

Comment:
@ashtul any complaints on it or you just spot it?

Reply:
@MeirShpilraien 
Mark requested we return 64 bits.
Tried a few other ways which you can see in other commits. but this is the simplest